src/core/lib/iomgr/timer_generic.cc

```diff
@@ -429,8 +429,9 @@ static void timer_init(grpc_timer* timer, grpc_millis deadline,
       note_deadline_change(shard);
       if (shard->shard_queue_index == 0 && deadline < old_min_deadline) {
 #if GPR_ARCH_64
-        // TODO: sreek - Using c-style cast here. static_cast<> gives an error
-        // (on mac platforms complaining that gpr_atm* is (long *) while
+        // TODO(sreek): Using c-style cast here. static_cast<> gives
+        // an error (on mac platforms complaining that gpr_atm* is (long *)
+        // while
         // (&g_shared_mutables.min_timer) is a (long long *). The cast should be
```

Comment:
Nit: Please fix line wrapping in this comment.

Reply:
Done